enterprise_subsidy/urls.py

```diff
@@ -57,15 +57,13 @@
     re_path(r'^admin/clearcache/', include('clearcache.urls')),
     re_path(r'^admin/', admin.site.urls),
     re_path(r'^api/', include(api_urls)),
-    re_path(r'^api-docs(?P<format>\.json|\.yaml)$', schema_view.without_ui(cache_timeout=0), name='schema-json'),
-    re_path(r'^api-docs/$', schema_view.with_ui('swagger', cache_timeout=0), name='schema-swagger-ui'),
     re_path(r'^auto_auth/$', core_views.AutoAuth.as_view(), name='auto_auth'),
     re_path(r'', include('csrf.urls')),  # Include csrf urls from edx-drf-extensions
     re_path(r'^health/$', core_views.health, name='health'),
     # DRF-spectacular routes below
     # TODO: deprecate the `/api-docs/` route in favor of the routes below.
+    re_path(r'^(?:api-docs/|api/schema/swagger-ui/)$', spec_swagger_view.as_view(url_name='schema'), name='swagger-ui'),
     path('api/schema/', spectacular_view.as_view(), name='schema'),
-    path('api/schema/swagger-ui/', spec_swagger_view.as_view(url_name='schema'), name='swagger-ui'),
```

Comment:
Hm, I don't quite think we want to actually change the location of the new swagger, since it's consistent with the new redoc location.  Instead, I'd advocate we change the LOGIN_REDIRECT_URL to the new location: https://github.com/openedx/enterprise-subsidy/blob/80ebc65d52c86f714bdbabf5c3c6727931d11b3a/enterprise_subsidy/settings/base.py#L331

Reply:
you could also put in a 301 redirect from the old URL (`/api-docs/`) to the new location in case people still have it bookmarked.